pyw3d/objects.py

```diff
@@ -839,10 +941,20 @@ def blend(self):
             #TODO
         blender_object.game.physics_type = 'DYNAMIC'
         blender_object.game.use_ghost = True
-
-        self.apply_material(blender_object)
-        blender_object.layers = [layer == 0 for layer in range(20)]
```

Comment:
This line should *always* be applied to any Blender object we want viewers to see.